olp-cpp-sdk-core/src/cache/DefaultCache.cpp

```diff
@@ -332,10 +332,9 @@ boost::optional<std::pair<std::string, time_t>> DefaultCache::GetFromDiscCache(
     const std::string& key) {
   if (protected_cache_) {
     auto result = protected_cache_->Get(key);
-
     if (result) {
-      return std::make_pair(std::move(result.value()),
-                            std::numeric_limits<time_t>::max());
+      auto default_expiry = KeyValueCache::kDefaultExpiry;
+      return std::make_pair(std::move(result.value()), default_expiry);
```

Comment:
Why not `return std::make_pair(std::move(result.value()), KeyValueCache::kDefaultExpiry);`?

Reply:
As discussed, this is not possible with std::make_pair() taking rvalue parameters. Because of the `std::move(result.value())` the second parameter is also expected to be an rvalue that cannot apply to constexpr.

Reply:
Thanks for the explanation.